qt/Session.cc

```diff
@@ -69,59 +52,18 @@ namespace
     for (const tr_quark key: keys)
       tr_variantListAddQuark (list, key);
   }
-}
-
-/***
-****
-***/
-
-void
-FileAdded::executed (int64_t tag, const QString& result, tr_variant * arguments)
-{
-  if (tag != myTag)
-    return;
-
-  if (result == QLatin1String ("success"))
-    {
-      tr_variant * dup;
-      const char * str;
-      if (tr_variantDictFindDict (arguments, TR_KEY_torrent_duplicate, &dup) &&
-          tr_variantDictFindStr (dup, TR_KEY_name, &str, NULL))
-        {
-          const QString myFilename = QFileInfo (myName).fileName ();
-          const QString name = QString::fromUtf8 (str);
-          QMessageBox::warning (qApp->activeWindow (),
-                                tr ("Add Torrent"),
-                                tr ("<p><b>Unable to add \"%1\".</b></p><p>It is a duplicate of \"%2\" which is already added.</p>").arg (myFilename).arg (name));
-        }
-
-      if (!myDelFile.isEmpty ())
-        {
-          QFile file (myDelFile);
-          file.setPermissions (QFile::ReadOwner | QFile::WriteOwner);
-          file.remove ();
-        }
-    }
-  else
-    {
-      QString text = result;
 
-      for (int i=0, n=text.size (); i<n; ++i)
-        if (!i || text[i-1].isSpace ())
-          text[i] = text[i].toUpper ();
-
-      QMessageBox::warning (qApp->activeWindow (),
-                            tr ("Error Adding Torrent"),
-                            QString::fromLatin1 ("<p><b>%1</b></p><p>%2</p>").arg (text).arg (myName));
-    }
+  /*
+   * If this object is passed as "ids" (compared by address), then recently active torrents are queried.
+   */
+  const QSet<int> recentlyActiveIds = QSet<int>::fromList (QList<int> ({ -1 }));
```

Comment:
I think this increases minimum required Qt version from currently used 4.6.2 due to the use of initializer list. Let's change to `QSet<int>() << -1` for now.